osCam/core/views.py

```diff
@@ -119,14 +130,21 @@ def rescaleFrame(frame, scale):
 
 @login_required
 def home(request):
-    return render(request, 'core/home.html')
+	
+    return render(request, 'core/home.html', {
+            "update_path_dialog":True,
+            "path":'/usr/media/uploads/',
+            "nextpaths":['videos', 'thumbnails']
+            })
+
+
 
 def gen(camera):
 	while True:
-		frame = camera.get_frame()
+		# frame = camera.get_frame()
+		frame = b'test'
```

Comment:
where is b'test' coming from?

Reply:
geeze great catch haha

Reply:
I still see this problem in the code? If it is resolved, shouldn't I see the change reflected in the code?